static/js/services/enketo.js

```diff
@@ -254,12 +254,25 @@ angular.module('inboxServices').service('Enketo',
         });
     };
 
+    var attachContent = function(doc, content) {
+      doc._attachments = doc._attachments || {};
+      doc._attachments.content = {
+        content_type: 'application/xml',
+        data: btoa(content)
+      };
+    };
+
     var update = function(formInternalId, record, docId) {
       // update an existing doc.  For convenience, get the latest version
       // and then modify the content.  This will avoid most concurrent
       // edits, but is not ideal.
       return DB().get(docId).then(function(doc) {
-        doc.content = record;
+        // previously XML was stored in the content property
+        // TODO delete this and other "legacy" code support commited against
```

Comment:
Hmm that's likely to be lost forever... Can you create an issue and link it here maybe, so that one day the issue reappears? Don't know if that's more likely.